distance3d/gjk/_gjk_original.py

```diff
@@ -5,7 +5,11 @@
 EPSILON = 10.0 * np.finfo(float).eps
 
 
-def gjk_distance_original(collider1, collider2):
+def gjk_distance_iterations(collider1, collider2):
+    return gjk_distance_original(collider1, collider2, return_interations=True)
```

Comment:
We don't need a additional argument for that. You can just always return the number of iterations and cut it off here:

```python
return gjk_distance_original(collider1, collider2, return_interations=True)[:-1]
```